capstone/src/main/java/com/google/univiz/api/SearchData.java

```diff
@@ -1,4 +1,18 @@
 package com.google.univiz.api;
 
+import com.google.auto.value.AutoValue;
+
 /** SearchData represents the college data specific to the Search feature */
-public class SearchData {}
+@AutoValue
+public abstract class SearchData {
+  private String collegeName;
+  private int collegeId;
+
+  public static SearchData create(String collegeName, int collegeId) {
+    return new AutoValue_SearchData(collegeName, collegeId);
+  }
```

Comment:
```suggestion

```
you don't need this, just the accessor methods is fine.

Reply:
I don't think I need the private variables at the beginning of the class, but don't I still need a create the SearchData obj?